generators/server/templates/src/main/java/package/service/_UserService.java

```diff
@@ -215,13 +215,19 @@ public User createUser(UserDTO userDTO) {
      * @param lastName last name of user
      * @param email email id of user
      * @param langKey language key
+     <%_ if (databaseType == 'mongodb' || databaseType == 'sql') { _%>
+     * @param imageUrl image URL of user
+     <%_ } _%>
      */
-    public void updateUser(String firstName, String lastName, String email, String langKey) {
+    public void updateUser(String firstName, String lastName, String email, String langKey<% if (databaseType == 'mongodb' || databaseType == 'sql') { %>, String imageUrl<% } %>) {
```

Comment:
please use `===` and `!==` for conditions. We might have old conditions without it but we are trying to change that